content/docs/getting-started/connect-dbaas.md

```diff
@@ -64,3 +65,26 @@ Configure `cURL` with DBaaS credentials.
 ```sh
 alias curl="curl -s -k -u \"${NUODB_CP_USER}:${NUODB_CP_PASSWORD}\""
 ```
+
+### Setting up the Terraform Provider
+
+To use terraform to manage your databases, you will want to use the [`nuodbaas` provider](https://registry.terraform.io/providers/nuodb/nuodbaas).
+The [provider documentation](https://registry.terraform.io/providers/nuodb/nuodbaas/latest/docs#schema) covers all of the available attributes.
+All of the them are optional.

```

Comment:
The user, password, and URL are not optional. They can be resolved from environment variables, but if nothing is resolved an error is thrown.

Reply:
They are optional in that the provider does not need them explicitly defined. I will clarify.

Reply:
Nit: "the them" -> "them"

Reply:
Good catch